wallet-webapps-common/src/main/webapp/vue-app/components/WalletImportKeyDrawer.vue

```diff
@@ -36,28 +38,37 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           ">
           <v-text-field
             v-model="walletPrivateKey"
-            :append-icon="walletPrivateKeyShow ? 'mdi-eye' : 'mdi-eye-off'"
             :rules="[rules.priv]"
-            :type="walletPrivateKeyShow ? 'text' : 'password'"
+            :type="password"
```

Comment:
shouldn't we keep the verification of the private key value by displaying/hiding the content ?

Reply:
discussed with @mzorai  to make it visible

Reply:
Ok then the private key will be always hidden, and not visible